AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/sample.json

```diff
@@ -17,5 +17,14 @@
 		 ]
 	}
      ]
```

Comment:
This file is not valid JSON.

Reply:
Thanks for the catch. I've reverted the PR and will let the sample owner know about the issue.